### PR TITLE
Fix donut chart if ratio between first and second value is very small

### PR DESCRIPTION
**Pull request by hansmaad**
Fixes https://github.com/gionkunz/chartist-js/issues/527 "donut chart not rendered correctly if ratio between first and second value is very small"

### CODE REVIEW

File: test/.jshintrc [hunk 1]
```diff
@@ -26,9 +26,11 @@
     "beforeEach": false,
     "browser": false,
     "describe": false,
+    "fdescribe": false,
```
Comment (gionkunz) [inline test/.jshintrc:29]: Why do we need those?

**Comment by gionkunz** [comment]
Hi Hannes, thanks for the awesome PR!

File: test/.jshintrc (hunk 1 above)
Comment (hansmaad) [inline test/.jshintrc:29]: To focus on a single `describe` or `it`. Helpful if you're working on a single feature and you don't want to run thousands of unrelated tests.

Comment (gionkunz) [inline test/.jshintrc:29]: Ah, so you change it back to describe and it if you're done with coding. Cool, then that makes sense.

**Comment by gionkunz** [comment]
You're becoming MVC :-) Most valuable contributor ;-)

**Pull request merged**